prompt2model/dataset_retriever/column_selection_prompt.py

```diff
@@ -0,0 +1,95 @@
+"""Utilities to construct an LLM "metaprompt" for our column selection."""
+
+from __future__ import annotations  # noqa FI58
+
+import json
+
+METAPROMPT_BASE = """Your objective is to carefully analyze the task, with the dataset mentioned and assign the columns into relevant input, output, ambigous and irrelevant columns for the given task. There should be at most one output column. It is possible to have no relevant columns, in which case return the input and output column as empty lists.  Answer in a json format, with the following keys: input, output, irrelevant, ambiguous"""  # noqa: E501
```

Comment:
I did some wording changes and fixed a typo in "ambiguous".
```suggestion
METAPROMPT_BASE = """Your objective is to carefully analyze the task and the dataset mentioned, and decide whether the columns are relevant input, relevant output, irrelevant for the given task, or if it is ambiguous. There should be at most one output column. It is possible to have no relevant columns, in which case return the input and output column as empty lists.  Answer in a json format, with the following keys: input, output, irrelevant, ambiguous"""  # noqa: E501
```